cms/static/js/views/video_transcripts.js

```diff
@@ -211,6 +252,39 @@ define(
                 this.renderMessage($transcriptContainer, 'failed', errorMessage);
             },
 
+            removeTranscript: function(event) {
+                var self = this,
+                    $transcriptEl = $(event.target).parents('.show-video-transcript-content'),
+                    languageCode = $transcriptEl.attr('data-language-code'),
+                    transcriptDeleteUrl = this.getTranscriptDeleteUrl(
+                        this.edxVideoID,
+                        languageCode,
+                        this.videoTranscriptSettings.transcript_delete_handler_url
+                    );
+
+                ViewUtils.confirmThenRunOperation(
+                    gettext('Are you sure you want to remove this transcript from the video?'),
+                    gettext('Removing a transcript from this video will have impact on all the video components using this video.'),  // eslint-disable-line max-len
```

Comment:
If you remove this transcript, the transcript will not be available for any components that use this video.